backend/de.metas.purchasecandidate.base/src/main/sql/postgresql/system/5760420_C_PurchaseCandidate_filter_by_DateOrdered.sql

```diff
@@ -0,0 +1,206 @@
+-- Column: C_PurchaseCandidate.IsRequisitionCreated
+-- 2025-07-09T07:31:12.362Z
+UPDATE AD_UI_Element SET IsDisplayedGrid='Y', SeqNoGrid=120,Updated=TO_TIMESTAMP('2025-07-09 07:31:12.362000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_UI_Element_ID=577949
+;
+
+-- UI Element: Bestelldisposition(540375,de.metas.purchasecandidate) -> Bestelldisposition(540894,de.metas.purchasecandidate) -> main -> 20 -> flags.Disponiert
+-- Column: C_PurchaseCandidate.IsPrepared
+-- 2025-07-09T07:31:12.368Z
+UPDATE AD_UI_Element SET IsDisplayedGrid='Y', SeqNoGrid=130,Updated=TO_TIMESTAMP('2025-07-09 07:31:12.368000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_UI_Element_ID=552268
+;
+
+-- UI Element: Bestelldisposition(540375,de.metas.purchasecandidate) -> Bestelldisposition(540894,de.metas.purchasecandidate) -> main -> 20 -> org.Sektion
+-- Column: C_PurchaseCandidate.AD_Org_ID
+-- 2025-07-09T07:31:12.373Z
+UPDATE AD_UI_Element SET IsDisplayedGrid='Y', SeqNoGrid=140,Updated=TO_TIMESTAMP('2025-07-09 07:31:12.373000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_UI_Element_ID=549349
+;
+
+-- UI Element: Geschäftspartner Pharma(540409,U) -> Bestell Schema(541103,de.metas.purchasecandidate) -> main -> 10 -> default.Lead time (days)
+-- Column: C_BP_PurchaseSchedule.LeadTimeOffset
+-- 2025-07-09T08:09:06.065Z
+DELETE FROM AD_UI_Element WHERE AD_UI_Element_ID=552339
+;
+
+-- 2025-07-09T08:09:06.067Z
+DELETE FROM AD_Element_Link WHERE AD_Field_ID=565044
+;
+
+-- Field: Geschäftspartner Pharma(540409,U) -> Bestell Schema(541103,de.metas.purchasecandidate) -> Lead Time Offset
+-- Column: C_BP_PurchaseSchedule.LeadTimeOffset
+-- 2025-07-09T08:09:06.072Z
+DELETE FROM  AD_Field_Trl WHERE AD_Field_ID=565044
+;
+
+-- 2025-07-09T08:09:06.078Z
+DELETE FROM AD_Field WHERE AD_Field_ID=565044
+;
+
+-- 2025-07-09T08:09:06.080Z
+/* DDL */ SELECT public.db_alter_table('C_BP_PurchaseSchedule','ALTER TABLE C_BP_PurchaseSchedule DROP COLUMN IF EXISTS LeadTimeOffset')
+;
+
+-- Column: C_BP_PurchaseSchedule.LeadTimeOffset
+-- 2025-07-09T08:09:06.108Z
+DELETE FROM  AD_Column_Trl WHERE AD_Column_ID=560223
+;
+
+-- 2025-07-09T08:09:06.113Z
+DELETE FROM AD_Column WHERE AD_Column_ID=560223
+;
+
+-- 2025-07-09T08:30:08.503Z
+DELETE FROM AD_Element_Link WHERE AD_Field_ID=564403
+;
+
+-- Field: Import Geschäftspartner(172,D) -> Import - Geschäftspartner(441,D) -> Lead Time Offset
+-- Column: I_BPartner.LeadTimeOffset
+-- 2025-07-09T08:30:08.507Z
+DELETE FROM  AD_Field_Trl WHERE AD_Field_ID=564403
+;
+
+-- 2025-07-09T08:30:08.515Z
+DELETE FROM AD_Field WHERE AD_Field_ID=564403
+;
+
+-- 2025-07-09T08:30:08.517Z
+/* DDL */ SELECT public.db_alter_table('I_BPartner','ALTER TABLE I_BPartner DROP COLUMN IF EXISTS LeadTimeOffset')
```

Comment:
pls extract the column dropping to a separate script.
before dropping pls take a backup

Reply:
done